internal/io/sql/stmt.go

```diff
@@ -8,17 +8,17 @@ import (
 // for each colName.
 func Insert(table string, colNames []string) string {
 	buf := bytes.NewBuffer(nil)
-	buf.WriteString("INSERT INTO ")
+	buf.WriteString("INSERT INTO \"")
 	buf.WriteString(table)
-	buf.WriteString("(")
+	buf.WriteString("\" (")
```

Comment:
This seems to address table names with whitespace in them but does it do anything to address column names with whitespace in them?